src/script/ceph-backport.sh

```diff
@@ -1082,7 +1082,7 @@ function try_known_milestones {
         mimic) mn="11" ;;
         nautilus) mn="12" ;;
         octopus) mn="13" ;;
-        pacific) mn="TBD" ;;
+        pacific) mn="14" ;;
```

Comment:
we need to be careful with this one - the milestone number is set by GitHub, pacific is _expected_ to be milestone number 14, but we need to verify that before setting it here

Also, the script will work for pacific even when the number is set to "TBD" - it will fall back to querying the GitHub API to get the number.

Reply:
ah, the "pacific" milestone has not been created yet, so we should not make this change now

it will be made when backporting to pacific starts.